src/finufft.cpp

```diff
@@ -1034,7 +1038,7 @@ int FINUFFT_EXEC(FINUFFT_PLAN* p, CPX* cj, CPX* fk){
 
 
 // DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
-int FINUFFT_DESTROY(FINUFFT_PLAN* p)
+int FINUFFT_DESTROY(FINUFFT_PLAN p)
```

Comment:
I feel a safety check that p!=NULL should be done before the rest of the destroy.

Reply:
I'm not sure this would catch all that much. If it points to some random address, it's going to segfault anyhow. We can adda  check, though.